src/qt/overviewpage.h

```diff
@@ -61,11 +62,11 @@ public Q_SLOTS:
     void updateStakeReportNow();
     void updateStakeReportbalanceChanged(CAmount, CAmount, CAmount, CAmount, CAmount, CAmount, CAmount);
     void setVotingStatus(QString text);
-
+    void on_pushButton_clicked();
```

Comment:
@proletesseract could we maybe use another name for this method? it reads very generic and does not refer to its real meaning

Reply:
Sure, that was just what was autogenerated by QT Creator. I'll commit the change when i get a chance.